module_utils/network_lsr/argument_validator.py

```diff
@@ -1547,6 +1572,15 @@ def _validate_post(self, value, name, result):
                         "but is '%s'" % result["mac"],
                     )
 
+            if result.get("match"):
```

Comment:
Just wondering, is `.get()` really necessary here or is the `match` key always present at this point? The other keys are expected to be always present here.

Reply:
Before, we may have match setting like `"match": None`, `.get()` is necessary, otherwise  following code will raise an error since `None` can not be enumerated.
```
            if "match" in result:
                if "path" in result["match"]:
```
but we normalize the `None` into `{}` in ArgvalidorDict now. `.get()` is not necessary, thanks for catching it.

Reply:
Changed into `if "match" in result:`